src/utils/front/focusElementWithHotkey.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Utility function to focus on an element by pressing a specified hotkey.
+ * @param target CSS selector for the element to be focused
+ * @param hotkey The key that should trigger the target element to be focused
+ */
+export const focusElementWithHotkey = (target: string, hotkey: string) => {
+  document.addEventListener('keydown', $event => {
+    if ($event.key !== hotkey || document.activeElement?.matches(target)) {
+      return
+    }
+    const targetElement = document.querySelector(target) as HTMLElement
+    targetElement?.focus()
+    $event.preventDefault()
+  })
```

Comment:
While `$event` won't hurt anything, all or at least most of our existing event listeners use `e` for the event's name.

Reply:
Sure thing. I've resolved this in dc3f8b5. Thanks!